test/python/expected/upgrade_validation/expected_dependency.out

```diff
@@ -1007,6 +1007,7 @@ View information_schema_tsql.views
 View sys.all_columns
 View sys.all_sql_modules_internal
 View sys.assembly_modules
+View sys.babelfish_configurations_view
```

Comment:
Let's not add more entries in this file. It means we don't have a proper upgrade tests for this view.